source/common/http/http1/codec_impl.h

```diff
@@ -56,6 +56,7 @@ class StreamEncoderImpl : public StreamEncoder,
   static const std::string LAST_CHUNK;
 
   ConnectionImpl& connection_;
+  void setIsContentLengthAllowed(bool value) { is_content_length_allowed_ = value; }
```

Comment:
nit: if you are going to bother having the setter, maybe assert that it's only called to go from true -> false and then remove the redundant setting to true above in the diff? TIOLI.

Reply:
+1

Reply:
Never mind.